### PR TITLE
Databunker

### PR DESCRIPTION
**Pull request by stremovsky**
- [x] I have referred to the list and did not find this submission already added.
- [x] I have read the contribution guidelines [here](https://github.com/humanetech-community/awesome-humane-tech/blob/main/contributing.md)
- [x] I have read the code of conduct and I agree to adhere to the same [here](https://github.com/humanetech-community/awesome-humane-tech/blob/main/code-of-conduct.md)
- [x] I have added my entry as the latest (last dotted point) under a topic, not at the beginning, not interleaved.

If your suggested entry is code project:

- [x] I have checked that the project is open source and has a [FSF-approved](https://www.gnu.org/licenses/license-list.html) license.

### Is there a similar software/tool etc., that already exists? Some cloud vendors offer commercial alternatives.

Yes

1. This product allows any company to host its own data in a secure way.
2. Companies do not need to reinvent the wheel and create their own secure storage to store customer data.
  
### Add any other topic/section where you wish to explain and elaborate
I think this product is great for humanity. It will minimize the cases of customer data exposure in case companies using Databunker will get hacked.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ Security is an important aspect of humane technology. It is also a vast field we
 - [LocalCDN](https://www.localcdn.org/) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/codeberg.svg?sanitize=true" width="16"/>](https://codeberg.org/nobody/LocalCDN) - A browser extension that emulates Content Delivery Networks to improve your online privacy.
 - [Universal Android Debloater](https://gitlab.com/W1nst0n/universal-android-debloater) - Debloat non-rooted smartphones, to improve privacy, security and battery life.
 - [RethinkDNS](https://www.rethinkdns.com) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/celzero/rethink-app) - DNS over HTTPS and Tor, DNSCrypt client, firewall and connection tracker for Android.
+- [Databunker](https://databunker.org/) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://databunker.org/) - Databunker is a network based GDPR compliant secure storage for personal data. It is an address book on steroids.
```
Comment (aschrijver) [inline README.md:128]: The github icon URL should point to the Github repository. And the list has the requirement that every entry must fit a single line. The part "It is an address book on steroids." is too long and should be removed.

**Comment by aschrijver** [comment]
Thanks for your entry @stremovsky an interesting project. As you can see there are some minor issues to fix.

One thing that is not entirely clear to me is if the solution can be self-hosted in its entirety, or that it always needs API access to your back-end SaaS. That might make it unsuitable for this particular list. Could you tell a bit more about that?

File: README.md (hunk 1 above)
Comment (stremovsky) [reply README.md]: Fixed

**Comment by stremovsky** [comment]
Hi, The solution is totally self-hosted.

I will review project documentation and add this to be more explicit.

**Comment by aschrijver** [comment]
Wonderful. Thanks for the changes. Project is great, and I will mention them on some of our community channels.

**Pull request merged**